### DISCUSSION TITLE
Where do you handle SIGPIPE?

### DISCUSSION THREAD

**Discussion by pengyj1984** [matt-42/lithium#108] [Q&A]
Hi matt: 
     The title is what I'm wondering? I did not see the code you handle SIGPIPE, or is there some other tricks to deal this?

**Comment by matt-42** [comment]
It is ignored. Because sendfile triggers sigpipe when the client close the connection before the end of the transfer and we don't want it to stop the server:
https://github.com/matt-42/lithium/blob/master/libraries/http_server/http_server/tcp_server.hh#L611

**Reply by pengyj1984** [reply]
Got it. Thanks for you answer.
